### ISSUE TITLE
Groups are deleted but membership isn't

### ISSUE THREAD

**Issue by thisismeonmounteverest** [BeWelcome/rox#80]
When a group gets deleted the code should remove the membership entries in the database as well but obviously fails to do so. This wasn't a problem with the old code but the new code uses the memberships as the entry point and not the groups so this leads to problems.

A database cleanup is needed and the group deletion process needs to be changed so that membership is really removed. (Use cascade in Doctrine entity class?)

**Comment by omoustaouda** [comment]
Hello.
I'd like to start contributing to BeWelcome,
is this a good task for starting?

**Comment by amnesiac84** [comment]
Hooray!
@thisismeonmounteverest can better judge if it's a good task for starting, but I'd say any open issue is a good task for starting ;)

**Comment by thisismeonmounteverest** [comment]
@omoustaouda: I don't think this would be a good starting point as it involves a fix for old code. Digging through that is a nightmare :)

But check #1 .
